py_zipkin/encoding/_helpers.py

```diff
@@ -47,17 +39,17 @@ def __init__(
         name,
         parent_id,
         span_id,
+        kind,
         timestamp,
         duration,
-        annotations,
-        tags,
-        kind,
         local_endpoint=None,
-        service_name=None,
-        sa_endpoint=None,
-        report_timestamp=True,
+        remote_endpoint=None,
+        debug=False,
```

Comment:
better to leave these boolean out vs serialize false

Reply:
These are just the defaults for the contructor. The decoder drops them if they're not True.